### PR TITLE
Add:archlinuxcn.md

### PR DESCRIPTION
**Pull request by CAICAIIs**
Translate document from Chinese to English and move to 'i18n\en\docusaurus-plugin-content-docs' folder

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+The Arch Linux Chinese Community Repository, an unofficial software repository driven by the Arch Linux Chinese community, includes many additional packages not provided by the official repositories, as well as git versions and other variants of existing software. Some of the packaging scripts for these packages originate from the AUR (Arch User Repository), but there are also many packages that differ from those in the AUR.
+
+## Arch Linux CN Software Source Replacement
+
+:::caution
+**To avoid problems when replacing software source configuration files, please make a backup of the system's default software source configuration files before proceeding.**
+:::
+
+1. Edit the `/etc/pacman.conf file` add the following line to the top of the file:
```
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:17]: `/etc/pacman.conf file` -> `/etc/pacman.conf`

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+```shell varcode
+[ ] (root) Are you the root user?
+---
+const SUDO = !root ? 'sudo ' : '';
+---
+${SUDO}pacman -Syyuu
+```
+
+##One-click source switching
```
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:67]: Missing whitespace

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+:::caution
+This method is only applicable for switching from the official source to our site's source. If you have already switched sources, please do not use the following commands.
+:::
+
+Use the `echo`  command to add the address of the current mirror to the end of the software source configuration file:
```
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:73]: Extra whitespace

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+---
+const SUDO = !root ? 'sudo ' : '';
+---
+${SUDO}echo -e "[archlinuxcn]\nServer = ${_http}://${_domain}/archlinuxcn/\$arch" >> /etc/pacman.conf
+```
+
+## References
+
+1. [ZMirror](https://mirrors.cernet.edu.cn/about)   
+2.  [Help Repository](https://github.com/mirrorz-org/mirrorz-help)
```
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:86]: Extra whitespace

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md (hunk 3 above)
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:73]: > the address of the current mirror

How about changing the `address` to `URL`? Also `software source configuration file` to `configuration file`

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Arch Linux CN
+title: Arch Linux CN Software Repository Mirror Usage Guide
+cname: 'archlinuxcn'
+---
+
+## Introduction to Arch Linux CN and Software Management
+
+The Arch Linux Chinese Community Repository, an unofficial software repository driven by the Arch Linux Chinese community, includes many additional packages not provided by the official repositories, as well as git versions and other variants of existing software. Some of the packaging scripts for these packages originate from the AUR (Arch User Repository), but there are also many packages that differ from those in the AUR.
```
Comment (tttturtle-russ) [inline i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:9]: `community` -> `Community`

**Review by tttturtle-russ — CHANGES_REQUESTED** [review]
1. Keep the format consistent with origin file.
2. Also this pr introduces some compilation errors but I'm not sure why, need to figure it out.

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md (hunk 4 above)
Comment (CAICAIIs) [reply i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md:86]: Sorry, I didn't pay attention to this.

File: i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md (hunk 5 above)
Comment (CAICAIIs) [reply i18n/en/docusaurus-plugin-content-docs/current/archlinuxcn.md]: I have already made the modifications

**Comment by tttturtle-russ** [comment]
Content is good now. We just need to fix the compilation error.

**Comment by mudongliang** [comment]
@CAICAIIs Please use the formatted git commit message. @tttturtle-russ When you merge this PR, please use **Square Merge**

**Comment by CAICAIIs** [comment]
> @CAICAIIs Please use the formatted git commit message. @tttturtle-russ When you merge this PR, please use **Square Merge**
Thank you for the guidance. I am a beginner in this area and I am currently learning how to use it

**Comment by CAICAIIs** [comment]
> @CAICAIIs Please use the formatted git commit message. @tttturtle-russ When you merge this PR, please use **Square Merge**

Thank you for the guidance. I am a beginner in this area and I am currently learning how to use it

**Comment by mudongliang** [comment]
截屏2024-11-03 11 13 55

It's fine. For us, we can merge it as one commit. However, it's good for your future if you can write good git title and commit message.

**Comment by CAICAIIs** [comment]
> 截屏2024-11-03 11 13 55
> It's fine. For us, we can merge it as one commit. However, it's good for your future if you can write good git title and commit message.

You're right, I'm still unfamiliar with the operation. Thank you for pointing it out.

**Comment by CAICAIIs** [comment]
> 截屏2024-11-03 11 13 55
> It's fine. For us, we can merge it as one commit. However, it's good for your future if you can write good git title and commit message.

Can you check again? Thank you for your careful guidance.

**Comment by mudongliang** [comment]
You can check it yourself. The commits in this PR is kind of messy.

**Comment by mudongliang** [comment]
For the commit history, you can check and refer to #102 #103

**Comment by CAICAIIs** [comment]
> For the commit history, you can check and refer to #102 #103

Thank you, I am not yet able to handle my commit and am currently studying.

**Comment by mudongliang** [comment]
> > For the commit history, you can check and refer to #102 #103
> 
> Thank you, I am not yet able to handle my commit and am currently studying.

No worry, keep learning. There is always a working method - keep your translation, create a new branch, copy the translation and create a new PR :)

**Comment by tttturtle-russ** [comment]
> > > For the commit history, you can check and refer to #102 #103
> 
> > 
> 
> > Thank you, I am not yet able to handle my commit and am currently studying.
> 
> 
> 
> No worry, keep learning. There is always a working method - keep your translation, create a new branch, copy the translation and create a new PR :)

agree, the easiest way is to open a new pr. and you can check how the compilation bug occurred.

**Comment by mudongliang** [comment]
duplicate with #105

**Pull request closed without merging**